locales/en-US/experiments.ftl

```diff
@@ -197,7 +218,25 @@ universalsearchContributors1Title = Senior UX Designer
 universalsearchContributors2Title = Staff Engineer
 universalsearchContributors3Title = Senior Engineer
 universalsearchEolwarning = We will automatically disable the Universal Search experiment and report the results when it ends.
-testpilotNewsupdatesGeneralupdate1Title = Breaking news!
-testpilotNewsupdatesGeneralupdate1Content = We added this news feed to Test Pilot to tell you about the latest news and experiment improvements.
-testpilotNewsupdatesGeneralupdate2Title = Changes on the way
-testpilotNewsupdatesGeneralupdate2Content = Page Shot, Activity Stream, Tab Center, and Pulse will graduate from Test Pilot next week as we get ready for the next round of experiments. We'll follow up very soon to tell you where these experiments are going and how you can keep using them.
\ No newline at end of file
+voicefillDescription = Speech recognition comes to Firefox. Voice fill lets you speak to your favorite search engines.
+voicefillIntroduction = <p>Voice Fill uses artificial intelligence to interpret speech input. As a participant in the Voice Fill experiment, your voice searches will teach our AI to provide smarter results.</p> <p>We've added Voice Fill to Google, Yahoo and DuckDuckGo search pages, and will expand to more sites as the experiment progresses.</p>
+voicefillMeasurements0 = When you choose to use Voice Fill, your recordings will be sent to our servers for processing to provide you with suggested text.
+voicefillMeasurements1 = Voice Fill will collect technical information related to the performance of the feature, such as how long it takes for our servers to process a submitted recording and whether you selected or modified any of the suggestions we gave.
+voicefillMeasurements2 = Voice Fill will also collect information about your use of the feature, such as how frequently Voice Fill is used and whether you used Voice Fill with specific tabs or pages such as Google, DuckDuckGo, or Yahoo.
+voicefillMeasurements3 = To help us improve the accuracy of our suggestions, we may retain a copy of submitted audio recordings, and related suggestion information. Audio recordings kept for this purpose will not be associated with any other personal identifiers.
+voicefillDetails0Copy = Look for the microphone icon on Yahoo, DuckDuckGo, and Google.
+voicefillDetails1Copy = Before using Voice Fill, you will need to allow voice input
+voicefillDetails2Copy = Voice Fill may offer you multiple options. Select the one you want to help train the AI.
+voicefillToursteps0Copy = Look for the Voice Fill icon when you use Google, Yahoo or DuckDuckGo.
```

Comment:
I can never make sense of the Yahoo! style guide to remember if they like the exclamation or not. #branding
Same w/ `voicefillDetails0Copy` and `voicefillMeasurements2` snippets above.

Bonus points if we want to order them the same, but I'm mildly OCD, so maybe it only afflicts me:
- Yahoo, DuckDuckGo, and Google
- Google, Yahoo or DuckDuckGo
- Google, DuckDuckGo, or Yahoo